agent/startup.sh

```diff
@@ -103,6 +103,12 @@ fi
 
 export CATTLE_AGENT_IP=${CATTLE_AGENT_IP:-$DETECTED_CATTLE_AGENT_IP}
 
+if [ -n "$CATTLE_PHYSICAL_HOST_UUID" ]; then
```

Comment:
Does this need to be conditional?  Can you just always pass it and it might be a zero length string?